.meta/sinks/papertrail.toml

```diff
@@ -0,0 +1,34 @@
+[sinks.papertrail]
+title = "Papertrail"
+beta = true
+common = false
+delivery_guarantee = "best_effort"
```

Comment:
Why not `at_least_once`?

Reply:
This wraps our tcp sink which is best_effort.